src/utilities/spacing/demo/styles.scss

```diff
@@ -9,7 +9,7 @@
     border-radius: sizes.$radius-medium;
   }
 
-  & > div {
+  & > .demo-example {
```

Comment:
For handling any case where a wrapping `div` with `dir=` attribute needs to be added.